src/test/java/org/hyperledger/besu/ethereum/trie/verkle/SimpleVerkleTrieTest.java

```diff
@@ -25,6 +25,55 @@
 
 public class SimpleVerkleTrieTest {
 
+
+  @Test
+  public void testToDotTrieOneValueNoRepeatingEdges() {
+    SimpleVerkleTrie<Bytes32, Bytes32> trie = new SimpleVerkleTrie<>();
+    Bytes32 key = Bytes32.fromHexString("0x00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff");
+    Bytes32 value = Bytes32.fromHexString("0x1000000000000000000000000000000000000000000000000000000000000000");
+    trie.put(key, value);
+
+    System.out.println(trie.toDotTree());
+  }
+
+
+  @Test
+  public void testToDotTrieTwoValuesNoRepeatingEdges() {
+    SimpleVerkleTrie<Bytes32, Bytes32> trie = new SimpleVerkleTrie<Bytes32, Bytes32>();
+    Bytes32 key1 = Bytes32.fromHexString("0x00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff");
+    Bytes32 value1 = Bytes32.fromHexString("0x1000000000000000000000000000000000000000000000000000000000000000");
+    Bytes32 key2 = Bytes32.fromHexString("0x00112233445566778899aabbccddeeff00112233445566778899aabbccddee00");
+    Bytes32 value2 = Bytes32.fromHexString("0x0100000000000000000000000000000000000000000000000000000000000000");
+
+    trie.put(key1, value1);
+    trie.put(key2, value2);
+
+    System.out.println(trie.toDotTree());
```

Comment:
I think we are missing assert for the tests

And I will maybe move this meeting to another class in order to not mix the different kind of tests (trie logic and dot display)

Reply:
I have addressed the suggestion and moved the tests to another class, as you recommended. However, I'd like to mention that in the tests, there are now very long single-line strings. This is due to the fact that the expected trees can be quite large, and it's a result of Nodes (such as InternalNode, BranchNode, StemNode) to have NullNodes or NullLeafNodes in their children lists.

Reply:
I was more thinking to put the string in a ressource file and load it in the test to compare with the result 

you can load a file like that in the test 

```java
private String getResources(final String fileName) throws IOException {
    var classLoader = DotDisplayTest.class.getClassLoader();
    return new String(
        classLoader.getResourceAsStream(fileName).readAllBytes(), StandardCharsets.UTF_8);
  }
```

and put your file that contains a string in "src/test/resources/"